setup.cfg

```diff
@@ -30,8 +30,8 @@ install_requires =
     einops
     # fiona 1.5+ required for fiona.transform module
     fiona>=1.5
-    # kornia 0.5.11+ required for kornia.augmentation.PadTo
-    kornia>=0.5.11
+    # kornia 0.6.4+ required for kornia.contrib.CombineTensorPatches
+    kornia>=0.6.4
```

Comment:
Note that this is the latest version of Kornia, released only a couple weeks ago. I'm not opposed to requiring this, but it is quite new.

Reply:
Removed

Reply:
This is actually needed for `compute_padding` a function I recently added to kornia. I could rewrite the function here but the corrected `CombineTensorPatches` will be required for the prediction PR.